src/multivariate/mvtdist.jl

```diff
@@ -1,7 +1,13 @@
 # Multivariate t-distribution
 
 ## Generic multivariate t-distribution class
-
+"""
+Multivariate Student-T distribution support affine transformations:
+```julia
+d = GenericMvTDist(ν, μ, Σ)
+c + B * d    # == GenericMvTDist(d.df, B * μ + c, B * Σ * B')
+```
+"""
```

Comment:
That should be moved to `GenericMvTDist` I think? (And be part of a larger more detailed docstring, similar to other distributions - seems it is missing completely currently.)